cmd/registry-experimental/cmd/compute/descriptor.go

```diff
@@ -128,3 +140,59 @@ func (task *computeDescriptorTask) Run(ctx context.Context) error {
 	}
 	return core.SetArtifact(ctx, task.client, artifact)
 }
+
+// descriptorFromZippedProtos runs protoc and returns the results.
+func descriptorFromZippedProtos(ctx context.Context, name string, b []byte) (*descriptorpb.FileDescriptorSet, error) {
+	// create a tmp directory
+	root, err := ioutil.TempDir("", "registry-protos-")
+	if err != nil {
+		return nil, err
+	}
+	// whenever we finish, delete the tmp directory
+	defer os.RemoveAll(root)
+	// unzip the protos to the temp directory
+	_, err = core.UnzipArchiveToPath(b, root+"/protos")
+	if err != nil {
+		return nil, err
+	}
+	return generateDescriptorForDirectory(ctx, name, root)
+}
```

Comment:
The commentary here doesn't seem to add any new information for readers. Aside from the part about the function running protoc anyway. Should we even mention that downstream implementation detail?

Reply:
I suppose I'm writing for non-expert readers (or myself, as I keep track of what I'm doing). Since this is "experimental" code, I'm erring on the side of being explicit (at least here, anyway). Specifically about protoc, I do think that's a noteworthy and potentially surprising thing to happen within the scope of this function.

Reply:
Done, thanks!